sphinx_gherkindoc/writer.py

```diff
@@ -172,9 +169,8 @@ def ticket_url_or_tag(tag: str) -> str:
         If tag is a ticket, return an anonymous embedded hyperlink for it,
         else tag itself.
         """
-        url = url_parser(tag)
-        if url:
-            return f"`{tag} <{url}>`__"
+        if url_parser:
+            return f"`{tag} <{url_parser(tag)}>`__"
```

Comment:
Maybe it is the late hour, but this change I didn't expect...
The old code called the url parser on the tag to see if it was handled.
The new code seems to just say "Hey, you defined a URL parser, so you better always return a URL to use?"

Reply:
Thank you Doug, that's an important observation, as I misunderstood what was going on there. I'll fix that up right quick!